docs/sources/developers/plugins/metadata.md

```diff
@@ -7,30 +7,178 @@ aliases = ["/docs/grafana/latest/plugins/developing/plugin.json/"]
+| `tracing`       | boolean                 | false    | For data source plugins. If the plugin supports streaming.                                                                                                                                                                                                                        |
+
+## dependencies
+
+Plugin dependencies.
+
+### Properties
+
+| Property         | Type   | Required | Description                                                |
+|------------------|--------|----------|------------------------------------------------------------|
+| `grafanaVersion` | string | **true** | Required Grafana version for this plugin, e.g. `6.7.x`     |
+| `plugins`        | array  | false    | An array of required plugins on which this plugin depends. |
+
+## includes
+
+### Properties
+
+| Property     | Type    | Required | Description |
+|--------------|---------|----------|-------------|
+| `addToNav`   | boolean | false    |             |
+| `component`  | string  | false    |             |
+| `defaultNav` | boolean | false    |             |
+| `name`       | string  | false    |             |
+| `role`       | string  | false    |             |
+| `type`       | string  | false    |             |
+
+## info
+
+Meta data for the plugin. Some fields are used on the plugins page in Grafana and others on Grafana.com if the plugin is published.
+
+### Properties
+
+| Property      | Type              | Required | Description                                                                                                                   |
+|---------------|-------------------|----------|-------------------------------------------------------------------------------------------------------------------------------|
+| `logos`       | [object](#logos)  | **true** | SVG images that are used as plugin icons.                                                                                     |
+| `updated`     | string            | **true** | Date when this plugin was built.                                                                                              |
+| `version`     | string            | **true** | Project version of this commit, e.g. `6.7.x`                                                                                  |
+| `author`      | [object](#author) | false    | Information about the plugin author.                                                                                          |
+| `description` | string            | false    | Description of plugin. Used on the plugins page in Grafana and for search on grafana.com.                                     |
```

Comment:
Here "grafana.com" is lowercase. Elsewhere, "Grafana.com" is uppercase. Reconcile throughout.